graphql/documents/data/config.graphql

```diff
@@ -60,6 +60,9 @@ fragment ConfigInterfaceData on ConfigInterfaceResult {
   autostartVideo
   autostartVideoOnPlaySelected
   continuePlaylistDefault
+  alwaysStartFromBeginning
+  trackActivity
+  ignoreInterval
```

Comment:
Given that these are exclusively used in the UI, we should consider moving them into the UI properties, instead of adding more backend config handles.

Reply:
Where is the UI properties file you want this to be moved to?